.github/workflows/pr-label-milestone-check.yml

```diff
@@ -0,0 +1,27 @@
+name: PR Label and Milestone Check
+
+on:
+  pull_request:
+    types: [opened, edited, labeled, unlabeled, synchronize]
+
+jobs:
+  check_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

Comment:
Should be using pinned commit action here:

`uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3`